src/main/java/org/openRealmOfStars/game/States/PlanetView.java

```diff
@@ -315,39 +298,36 @@ public PlanetView(final Planet planet, final boolean interactive,
     panel = new SpaceGreyPanel();
     panel.setLayout(new BoxLayout(panel, BoxLayout.Y_AXIS));
     credProd = new IconLabel(null, Icons.getIconByName(Icons.ICON_CREDIT),
-        ": " + planet.getTotalProduction(Planet.PRODUCTION_CREDITS));
+        ": 0");
     credProd.setToolTipText("Total production of credits");
     credProd.setAlignmentX(Component.LEFT_ALIGNMENT);
     panel.add(credProd);
 
     maintenance = new IconLabel(null,
-        Icons.getIconByName(Icons.ICON_MAINTENANCE),
-        ": " + planet.getMaintenanceCost());
+        Icons.getIconByName(Icons.ICON_MAINTENANCE), ": 0");
     maintenance.setToolTipText("Maintenance cost of planet");
     maintenance.setAlignmentX(Component.LEFT_ALIGNMENT);
     panel.add(maintenance);
 
     taxPanel = new WorkerProductionPanel(
         GameCommands.COMMAND_MINUS_TAX, GameCommands.COMMAND_PLUS_TAX,
-        Icons.ICON_TAX, ": " + planet.getTax(),
+        Icons.ICON_TAX, ": 0",
         "How many productions are converted to credits", listener);
     taxPanel.setAlignmentX(Component.LEFT_ALIGNMENT);
     taxPanel.setInteractive(interactive);
     panel.add(taxPanel);
 
-    metal = new IconLabel(null, Icons.getIconByName(Icons.ICON_METAL),
-        ": " + planet.getMetal());
+    metal = new IconLabel(null, Icons.getIconByName(Icons.ICON_METAL), ": 0");
```

Comment:
Total metal might not fit into one zero. I had planet with 54 metal on surface and even the 4 wasn't fully visible. I think this should have four zeroes just to be safe.